packages/builder/builder-rspack-provider/src/types/rspack.ts

```diff
@@ -1,6 +1,9 @@
-import type { Compiler, RspackOptions } from '@rspack/core';
+import type * as rspack from '@rspack/core';
 
-export { Compiler };
+type Compiler = rspack.Compiler;
+type RspackOptions = rspack.RspackOptions;
+
+export type { Compiler, rspack };
```

Comment:
Makybe the `rspack` type should be `Rspack` (In PascalCase), the `webpack` type is a special case.

Reply:
done